Command/AlignCommand.php

```diff
@@ -147,10 +166,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 }
                 $progress->advance();
             }
-            
+
             if ($flush) {
                 $output->writeln("");
             }
+
+            $scroll_id = $arrayElasticsearchIndex['_scroll_id'];
+            $arrayElasticsearchIndex = $this->client->scroll([
+                "scroll_id" => $scroll_id,  //...using our previously obtained _scroll_id
```

Comment:
use single quotes. And don't create a variable $scrollId, call $arrayElasticsearchIndex['_scroll_id'] immediatly here. 
Also no use for these comments